src/graphic/Fast3D/gfx_pc.h

```diff
@@ -118,9 +119,22 @@ struct RawTexMetadata {
 };
 
 struct ShaderMod {
-    bool enabled = false;
-    int16_t id;
-    uint8_t type;
+    const char* vertex;
```

Comment:
These can be strings since they're always returned as strings anyways. I think that means you can default `operator==`, as discussed earlier.